pkg/controller/options.go

```diff
@@ -63,6 +63,6 @@ type Options struct {
 func (o Options) ForControllerRuntime() controller.Options {
 	return controller.Options{
 		MaxConcurrentReconciles: o.MaxConcurrentReconciles,
-		RateLimiter:             ratelimiter.NewController(o.GlobalRateLimiter),
+		RateLimiter:             ratelimiter.NewController(),
```

Comment:
The idea is each controller uses `NewController` to produce a per-item exponential backoff controller to rate limit reconciles that return `Requeue: true` or a non-nil `error`.

The `NewGlobal` reconciler is passed to `ratelimiter.NewReconciler` to rate limit _all_ reconciles.